uberpoet/blazeprojectgen.py

```diff
@@ -143,14 +147,24 @@ def gen_app(self, app_node, node_list, target_loc, loc_json_file_path):
             # Copy the LOC file into the generated project.
             shutil.copyfile(loc_json_file_path, join(self.app_root, basename(loc_json_file_path)))
 
+        serializable_module_index = {
+            key: {
+                "file_count": len(value["files"]),
```

Comment:
`files` in the original `module_index` is not currently serializable and does not buy us much to add support for it for now. This just prints out the number of files for each module.